test/functional/index.js

```diff
@@ -300,6 +300,39 @@ describe('Autocomplete', () => {
           expect(autocomplete.state.options[0]).to.equal('France')
           expect(autocomplete.state.query).to.equal('France')
         })
+
+        it('is prefilled with a simple custom inputValue template', () => {
```

Comment:
Is there a way to write these tests that makes it more explicit what's going on?

They currently feel like they're relying on a side effect of the way the `inputValue` template is implemented in each one, and whether it's dealing with a string or an object. It's not clear to me how the difference is described by the test description, or why the different functions should give different results.